docs/docker_prediction_request_client.md

```diff
@@ -0,0 +1,38 @@
+# Build Docker Images
+
+You can build your own docker image using the Dockerfile in the docker direcoty.
+```
+docker build -t tensorflow-serving-sidecar-client:latest .
```

Comment:
Getting the following error:

W: The repository 'http://ppa.launchpad.net/jonathonf/python-3.6/ubuntu xenial Release' does not have a Release file.
E: Failed to fetch http://ppa.launchpad.net/jonathonf/python-3.6/ubuntu/dists/xenial/main/binary-amd64/Packages  403  Forbidden
E: Some index files failed to download. They have been ignored, or old ones used instead.
The command '/bin/sh -c apt-get update' returned a non-zero code: 100

Reply:
**ppa:jonathonf was removed by the developer.**
https://launchpad.net/~jonathonf

You can refer to this discussion in the following link.
https://gist.github.com/monkut/c4c07059444fd06f3f8661e13ccac619#gistcomment-3112630


I have updated the Dockerfile and tested it. It's working now.

Reply:
Thanks for the update @fai555 .